chia/util/initial-config.yaml

```diff
@@ -515,10 +515,7 @@ wallet:
   full_node_peer:
     host: *self_hostname
     port: 8444
-  # The path of NFT off-chain metadata cache
-  nft_metadata_cache_path: "nft_cache"
-  # The length of NFT ID prefix will be used as hash index
-  nft_metadata_cache_hash_length: 3
+
```

Comment:
Just double checking, is this an intentional deletion? It seems unrelated to this PR.

Reply:
there's a lot of things to cleanup, but let's focus on clawback for this PR